### DISCUSSION TITLE
R-Ladies São Paulo - As metas foram atualizadas com sucesso!

### DISCUSSION THREAD

**Discussion by giscus** [R-Ladies-Sao-Paulo/blog#32] [General]
# R-Ladies São Paulo - As metas foram atualizadas com sucesso!

https://rladies-sp.org/posts/2024-01-planejamento-do-ano/

**Comment by claudiaanjos** [comment]
Conheci a comunidade há pouco tempo e já estou adorando todas as iniciativas. Obrigada por compartilhar conosco!
